web/gui/dashboard_info.js

```diff
@@ -3196,6 +3196,32 @@ netdataDashboard.context = {
         info: 'Difference between the number of process created and the number of threads created per period(<code>process</code> dimension), it also shows the number of possible zombie process running on system.'
     },
 
+    'ebpf.outbound_bytes': {
+        title : 'Outbound bytes',
+        info: 'Bytes sent to specific destination using TCP or UDP ports. It shows the first 50 outbound connections collected for the plugin, if there are more connections they will be bundled into the `other` dimension.'
```

Comment:
> It shows the first 50 outbound connections collected

> yo can increase the number of shown dimensions dimensions by changing the `maximum dimensions` setting.

It seems you cant hardcode a number in the `info`

Reply:
Good catch, I updated it now.